src/components/MarsApp.css

```diff
@@ -14,9 +14,9 @@
   -webkit-background-clip: text;
   font-weight: 10px;
   font-size: 50px;
-  font-family: fantasy;
+  font-family: fantasy, sans-serif, cursive;
   letter-spacing: 2px;
-  margin: 50px auto;
+  margin: 50px auto 50px auto;
```

Comment:
`margin: 50px auto;` and `margin: 50px auto 50px auto;` are equivalent aren't they? They both set top and bottom margins to `50px` and left and right margins to `auto`